### PR TITLE
Adapt to new course content

### LINKED ISSUE #1898: Adapt intro course landing page to the new content
The intro course now have a ["what is quantum"](https://learn.qiskit.org/course/introduction/what-is-quantum) section, so we have to remove the "what is quantum" prerequisite:
Captura de pantalla 2021-06-18 a las 8 56 51

and add the section to the course pages:
Captura de pantalla 2021-06-18 a las 8 57 20

### PR DESCRIPTION
**Pull request by y4izus**
Fix #1898

### CODE REVIEW

File: components/textbook-demo/CoursePagesSection.vue [hunk 1]
```diff
@@ -77,6 +77,12 @@ export default class CoursePagesSection extends Vue {
       segment: { action: `${this.segmentActionBase} the-atoms-of-computation` },
       url: 'https://platypus.qiskit.org/course/introduction/the-atoms-of-computation'
     },
+    {
+      image: 'the-atoms-of-computation-page-preview.png',
```
Comment (eddybrando) [inline components/textbook-demo/CoursePagesSection.vue:81]: This image doesn't seem to be related to the "What is quantum?" course.

**Review by eddybrando — CHANGES_REQUESTED** [review]
I believe the course image isn't right.

File: components/textbook-demo/CoursePagesSection.vue (hunk 1 above)
Comment (y4izus) [reply components/textbook-demo/CoursePagesSection.vue:81]: We don't have the final images yet :) https://github.com/Qiskit/qiskit.org/issues/1887

Comment (eddybrando) [reply components/textbook-demo/CoursePagesSection.vue:81]: Ok, got it. We'll update the image with that PR later. Thanks 🙂

**Review by eddybrando — APPROVED** [review]
